client/server/api/integration/index.js

```diff
@@ -69,6 +69,7 @@ describe( 'api', () => {
 			} );
 		} );
 
+		/* eslint-disable-next-line jest/expect-expect */
```

Comment:
Thanks for cleaning up eslint errors!

Instead of disabling the rule an alternative would be to configure `jest/expect-expect` to accept `localRequest.*.expect` by adding this at the top of the file:

```
/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "localRequest.*.expect"] }] */
```

(doc https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/expect-expect.md#assertfunctionnames)

Reply:
The trouble is that the `.expect()` assert doesn't even work 🙁 Just try to change one of the `.expect( 400 )` calls to `.expect( 200 )`. The tests still pass. I'd like to fix that in a followup.